docs/modules/ROOT/pages/object-storage/delete.adoc

```diff
@@ -0,0 +1,8 @@
+= Delete an Object Storage Bucket
+
+[source,bash]
+----
+$ oc delete objectbucket my-bucket
+----
+
+IMPORTANT: If you delete the objectBucket object from the cluster, it will delete the actual bucket as well!
```

Comment:
```suggestion
IMPORTANT: If you delete the `ObjectBucket` object from the cluster, it will delete the actual bucket as well!
```

Maybe also document the deletion policy on this page?

Reply:
I've added a note that it will be available later.